### PR TITLE
EWS O365 Playbooks Output Fix

### PR DESCRIPTION
**Pull request by darkushin**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/33856

## Description
Modified the context keys of the `SearchAction.Results` to a single key.

### CODE REVIEW

File: Packs/EWS/ReleaseNotes/1_8_2.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
+#### Playbooks
+##### O365 - Security And Compliance - Search Action - Delete
+Changed the context-keys mapping of the results returned for a `SearchAction` object. After the change, all the
+ results are mapped to **O365.SecurityAndCompliance.ContentSearch.SearchAction.Results**.
```
Comment (amshamah419) [inline Packs/EWS/ReleaseNotes/1_8_2.md:5]:
```suggestion
Changed the context-keys mapping of the results returned for a `SearchAction` object. After the change, all of the
 results are mapped to **O365.SecurityAndCompliance.ContentSearch.SearchAction.Results**.
```

File: Packs/EWS/ReleaseNotes/1_8_2.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+##### O365 - Security And Compliance - Search Action - Preview
+Changed the context-keys mapping of the results returned for a `SearchAction` object. After the change, all the
+ results are mapped to **O365.SecurityAndCompliance.ContentSearch.SearchAction.Results**.
```
Comment (amshamah419) [inline Packs/EWS/ReleaseNotes/1_8_2.md:8]:
```suggestion
Changed the context-keys mapping of the results returned for a `SearchAction` object. After the change, all of the
 results are mapped to **O365.SecurityAndCompliance.ContentSearch.SearchAction.Results**.
```

File: Packs/EWS/ReleaseNotes/1_8_2.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+##### O365 - Security And Compliance - Search
+- Changed the context-keys mapping of the results returned for a `SearchAction` object. After the change, all the
+ results are mapped to **O365.SecurityAndCompliance.ContentSearch.SearchAction.Results**.
+- Changed the context-keys mapping of the results returned for a `Search` object. After the change, all the
+ results are mapped to **O365.SecurityAndCompliance.ContentSearch.Search.Results**.
```
Comment (amshamah419) [inline Packs/EWS/ReleaseNotes/1_8_2.md:13]:
```suggestion
- Changed the context-keys mapping of the results returned for a `SearchAction` object. After the change, all of the
 results are mapped to **O365.SecurityAndCompliance.ContentSearch.SearchAction.Results**.
- Changed the context-keys mapping of the results returned for a `Search` object. After the change, all of the
 results are mapped to **O365.SecurityAndCompliance.ContentSearch.Search.Results**.
```

File: Packs/EWS/ReleaseNotes/1_8_2.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+##### O365 - Security And Compliance - Search Action - Delete
+Changed the context-keys mapping of the results returned for a `SearchAction` object. After the change, all of the
+ results are mapped to **O365.SecurityAndCompliance.ContentSearch.SearchAction.Results**.
+##### O365 - Security And Compliance - Search Action - Preview
+Changed the context-keys mapping of the results returned for a `SearchAction` object. After the change, all of the
+ results are mapped to **O365.SecurityAndCompliance.ContentSearch.SearchAction.Results**.
+##### O365 - Security And Compliance - Search
+- Changed the context-keys mapping of the results returned for a `SearchAction` object. After the change, all of the
+ results are mapped to **O365.SecurityAndCompliance.ContentSearch.SearchAction.Results**.
+- Changed the context-keys mapping of the results returned for a `Search` object. After the change, all of the
```
Comment (yuvalbenshalom) [inline Packs/EWS/ReleaseNotes/1_8_2.md:12]: need to add the note that dose are breaking changes

**Comment by glicht** [comment]
@darkushin can you add a playbook test for at least the search and preview

**Comment by darkushin** [comment]
> @darkushin can you add a playbook test for at least the search and preview

Working on it

File: Packs/EWS/Playbooks/playbook-O365-SecurityAndCompliance-Search.yml [hunk 5]
```diff
@@ -178,16 +178,9 @@ outputs:
...
-- contextPath: O365.SecurityAndCompliance.ContentSearch.Search.SuccessResults.ItemsCount
-  description: The number of security and compliance search results in the location.
-  type: Number
-- contextPath: O365.SecurityAndCompliance.ContentSearch.Search.SuccessResults.Size
-  description: The byte size of the security and compliance search results in the
-    location.
-  type: Number
+- contextPath: O365.SecurityAndCompliance.ContentSearch.Search.SuccessResults
+  description: Security and compliance search results.
+  type: unknown
```
[the remark this answers is not in the document]
Comment (darkushin) [reply Packs/EWS/Playbooks/playbook-O365-SecurityAndCompliance-Search.yml:183]: It's a list of dictionaries

image

**Review by yuvalbenshalom — APPROVED** [review]

File: Packs/EWS/Playbooks/playbook-O365-SecurityAndCompliance-Search.yml (hunk 5 above)
[the remark this answers is not in the document]
Comment (yuvalbenshalom) [reply Packs/EWS/Playbooks/playbook-O365-SecurityAndCompliance-Search.yml:183]: I agree with @darkushin this is an object

**Pull request merged**